pgctl/cli.py

```diff
@@ -457,6 +494,22 @@ def parser():
     return parser
 
 
+def _humanize_seconds(seconds):
+    for period_name, period_length in (
```

Comment:
I'd almost prefer something like `3 days 2 hours 5 minutes 3 seconds` instead of `3.1 days`